appveyor.yml

```diff
@@ -1,27 +1,48 @@
-branches:
-  only:
-    - master
+build: off
+deploy: off
 
 environment:
-  PATH: C:\Ruby%RUBY_VERSION%\DevKit\mingw\bin;C:\Ruby%RUBY_VERSION%\bin;C:\Ruby%RUBY_VERSION%\DevKit\bin;%PATH%
+  PATH: C:\Ruby%RUBY_VERSION%\bin;%PATH%
+  APPVEYOR_SAVE_CACHE_ON_ERROR: True
   matrix:
-  - RUBY_VERSION: "24-x64"
-  - RUBY_VERSION: "23-x64"
-  - RUBY_VERSION: "23"
-  - RUBY_VERSION: "22"
+  - RUBY_VERSION: _trunk
+  - RUBY_VERSION: 26-x64
+  - RUBY_VERSION: 25-x64
+  - RUBY_VERSION: 24-x64
+  - RUBY_VERSION: 23-x64
+  - RUBY_VERSION: 23
+  - RUBY_VERSION: 22
+
+init:
+  - ps: |
+      if ($env:RUBY_VERSION -eq '_trunk') {
+        $trunk_uri = 'https://ci.appveyor.com/api/projects/MSP-Greg/ruby-loco/artifacts/ruby_trunk.7z'
```

Comment:
Can you explain what this URL is for?

Reply:
It is my trunk/head build, it's been used by several gems/repos for Appveyor CI testing.  RubyGems, RDoc, Puma, EventMachine, etc...

Reply:
Okay, awesome. Just wanted to check that it wasn't some mistake or locally specific thing.